h2o-py/h2o/model/extensions/std_coef.py

```diff
@@ -4,12 +4,14 @@
 
 class StandardCoef:
 
-    def _std_coef_plot(self, num_of_features=None, server=False):
+    def _std_coef_plot(self, num_of_features=None, server=False, save_plot_path:str=None, **kwargs):
         """
         Plot a GLM model"s standardized coefficient magnitudes.
 
         :param num_of_features: the number of features shown in the plot.
         :param server: if true set server settings to matplotlib and show the graph
+        :param save_plot_path: a path to save the plot via using mathplotlib function savefig
+        :param **kwargs: can be used to pass in additional savefig configuration
```

Comment:
if the kwargs are forwarded to a specific function, then why not name it directly `**savefig`